operator-k8s/src/router.rs

```diff
@@ -0,0 +1,42 @@
+use axum::{Extension, Json};
+use flume::Sender;
+use operator_api::HeartbeatStatus;
+use prometheus::{Encoder, Registry};
+use tracing::error;
+
+/// metrics handler
+#[allow(clippy::unused_async)] // require by axum
+pub(crate) async fn metrics(Extension(registry): Extension<Registry>) -> String {
+    let mut buf1 = Vec::new();
+    let encoder = prometheus::TextEncoder::new();
+    let metric_families = registry.gather();
+    if let Err(err) = encoder.encode(&metric_families, &mut buf1) {
+        error!("failed to encode custom metrics: {}", err);
+        return String::new();
+    }
+    let mut res = String::from_utf8(buf1).unwrap_or_default();
+    let mut buf2 = Vec::new();
+    if let Err(err) = encoder.encode(&prometheus::gather(), &mut buf2) {
+        error!("failed to encode prometheus metrics: {}", err);
+        return String::new();
+    }
+    res.push_str(&String::from_utf8_lossy(&buf2));
+    res
+}
+
+/// healthz handler
```

Comment:
typo, it's "healthy"

Reply:
I guess it is not a typo.
FYI: https://stackoverflow.com/questions/43380939/where-does-the-convention-of-using-healthz-for-application-health-checks-come-f